best-practices/check_deprecated_apis.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: check-deprecated-apis
+  annotations:
+    policies.kyverno.io/category: Best Practices
+    policies.kyverno.io/description: >-
+      Kubernetes APIs are sometimes deprecated and removed after a few releases. 
+      As a best practice older API versions should be replaced with newer versions.
+spec:
+  validationFailureAction: enforce
+  rules:
+  - name: validate-v1-22-removals
+    match:
+      resources:
+        kinds:
+        - admissionregistration.k8s.io/v1beta1/ValidatingWebhookConfiguration
```

Comment:
Did you see [this issue](https://github.com/kyverno/kyverno/issues/2158#issuecomment-882023960)? Not sure this will do anything if accurate.

Reply:
A violation will still be shown in the policy reports, right?

Reply:
Good point, not sure.